ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/vlan-interfaces.yml

```diff
@@ -128,6 +128,26 @@ vlan_interfaces:
       10.10.64.150:
         source_interface: Loopback0
 
+  Vlan43:
+    description: SVI Description
+    shutdown: false
+    ipv6_address: a0::1/64
+    ipv6_dhcp_relay_destinations:
+      - address: a0::2
+        vrf: TEST
+        local_interface: Loopback44
+        link_address: a0::4
+
+  Vlan44:
+    description: SVI Description
+    shutdown: false
+    ipv6_address: a0::4/64
+    ipv6_dhcp_relay_destinations:
+      - address: a0::5
+        vrf: TEST
+        source_address: a0::6
+        link_address: a0::7
+
```

Comment:
Make one of the tests with two entries maybe ?

Reply:
Fixed, thanks!